app/models/provider_authorization.rb

```diff
@@ -157,10 +157,6 @@ def refresh_access_token!
        'Content-Type' => 'application/x-www-form-urlencoded'
      }
     }
-    if CONFIG.ca_path || CONFIG.ca_file
-      options[:ssl_ca_file] = CONFIG.ca_file
-      options[:ssl_ca_path] = CONFIG.ca_path
-    end
```

Comment:
I don't think this will cause any probs in production, but it might in local dev environments where the CA file might be somewhere other than the default, right? Not a blocker, but something to keep in mind.